flexget/plugins/operate/free_space.py

```diff
@@ -3,10 +3,14 @@
 from loguru import logger
 
 from flexget import plugin
+from flexget.config_schema import one_or_more
 from flexget.event import event
 
 logger = logger.bind(name='free_space')
 
+ABORT_UNDER = 'below'
+ABORT_OVER = 'above'
```

Comment:
Let's keep the naming consistent, either above/below, or under/over. I'm good with either one.